### ISSUE TITLE
build fails with "Error: no value for string: a11y.graphSetRadioButtonGroup"

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/calculus-grapher#375]
In https://github.com/phetsims/calculus-grapher/commit/242e14d6c01642545e9d81d8178b292e676ffd56, I factored out duplicate CalculusGrapherFluent identifiers in many source files. For example, in GraphSetRadioButtonGroup.ts:

```ts
const ACCESSIBLE_STRINGS = CalculusGrapherFluent.a11y.graphSetRadioButtonGroup;
...
     accessibleName: ACCESSIBLE_STRINGS.accessibleNameStringProperty,
      accessibleHelpText: ACCESSIBLE_STRINGS.accessibleHelpTextStringProperty,
```

Now builds are failing with the error shown below.  The unbuilt sim runs fine, and the strings that it’s complaining about are clearly shown in A11y View.

First noticed in CT, reproduced with a local build:

```
% cd calculus-grapher; grunt
Running “default” task

Running “lint-project”
Linting completed successfully.

Running “report-media”

Running “clean”

Running “build”
>> Type Check complete:			101ms
>> Transpile complete:			836ms
Building runnable repository (calculus-grapher, brands: phet)
Building brand: phet
>> Webpack build complete:		2075ms
/Users/cmalley/PhET/GitHub/chipper/js/common/ChipperStringUtils.ts:142
    throw new Error( `no value for string: ${key}` );
       ^

Error: no value for string: a11y.graphSetRadioButtonGroup
  at Object.getStringEntryFromMap (/Users/cmalley/PhET/GitHub/chipper/js/common/ChipperStringUtils.ts:142:15)
  at <anonymous> (/Users/cmalley/PhET/GitHub/chipper/js/grunt/getStringMap.ts:301:46)
  at Array.forEach (<anonymous>)
  at <anonymous> (/Users/cmalley/PhET/GitHub/chipper/js/grunt/getStringMap.ts:296:15)
  at Array.forEach (<anonymous>)
  at <anonymous> (/Users/cmalley/PhET/GitHub/chipper/js/grunt/getStringMap.ts:295:23)
  at Array.forEach (<anonymous>)
  at getStringMap (/Users/cmalley/PhET/GitHub/chipper/js/grunt/getStringMap.ts:170:24)
  at default (/Users/cmalley/PhET/GitHub/chipper/js/grunt/buildRunnable.ts:148:41)
  at async <anonymous> (/Users/cmalley/PhET/GitHub/chipper/js/grunt/tasks/build.ts:148:11)
```

**Comment by pixelzoom** [comment]
I removed the const from GraphSetRadioButtonGroup.ts:

```diff
-const ACCESSIBLE_STRINGS = CalculusGrapherFluent.a11y.graphSetRadioButtonGroup;

-      accessibleName: ACCESSIBLE_STRINGS.accessibleNameStringProperty,
-      accessibleHelpText: ACCESSIBLE_STRINGS.accessibleHelpTextStringProperty,
+      accessibleName: CalculusGrapherFluent.a11y.graphSetRadioButtonGroup.accessibleNameStringProperty,
+      accessibleHelpText: CalculusGrapherFluent.a11y.graphSetRadioButtonGroup.accessibleHelpTextStringProperty,
```

Now it’s complaining about the same pattern for a identifier in another file.  So this is definitely the problem. I'm guessing that build tools is looking for complete identifiers, and tossing any strings that it doesn't find. 

Please tell me this can be corrected in build and that I don’t have to revert to long identifiers with lots of duplication.
